backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/reportPriceListComparation.sql

```diff
@@ -121,6 +121,7 @@ FROM M_ProductPrice pp
     FROM report.Valid_PI_Item_Product_V vip
     -- WHERE isInfiniteCapacity = 'N' -- task 09045/09788: we can also export PiiPs with infinite capacity
     WHERE vip.M_Product_ID = pp.M_Product_ID
+      AND (vip.c_bpartner_id IS NULL OR $1 IS NULL OR vip.c_bpartner_id = $1)
```

Comment:
now, I noticed: pleas using named parameters; we are trying to get rid of $ parameters